internal/pkg/cli/pipeline_init.go

```diff
@@ -247,14 +247,36 @@ func (o *initPipelineOpts) createPipelineProvider() (manifest.Provider, error) {
 	return manifest.NewProvider(config)
 }
 
+func (o *initPipelineOpts) getEnvFromCache(environmentName string) (*config.Environment, error) {
```

Comment:
nit: Can we rename this method to `getEnvConfig`?

Reply:
I addressed.